pom.xml

```diff
@@ -1818,9 +1818,19 @@
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>${maven-javadoc-plugin.version}</version>
           <configuration>
+            <sourceFileExcludes>
+              <exclude>**/package-info.java</exclude>
+            </sourceFileExcludes>
             <doclint>none</doclint>
             <notimestamp>true</notimestamp>
           </configuration>
+          <dependencies>
+            <dependency>
+              <groupId>com.google.protobuf</groupId>
+              <artifactId>protobuf-java</artifactId>
+              <version>${grpc.protobuf-compile.version}</version>
+            </dependency>
+          </dependencies>
```

Comment:
Instead of adding protobuf as depenency, I think we should skip creating javadoc for generated code.  Please see HDDS-11142.